tools/openapi/src/openapi.js

```
@@ -0,0 +1,202 @@
+  /** @type {any[]} */
+  const out = [];
+
+  for (const pp of endpointSpec.pathParameters || []) {
+    out.push({
+      name: pp.name,
+      in: 'path',
+      required: true,
+      description: pp.description,
+      schema: mapParameterSchema(pp.type),
+    });
+  }
```

ESLint

[error] iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations.

(no-restricted-syntax)